### PR TITLE
use cdp build version

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

File: delivery.yaml [hunk 1]
```diff
@@ -17,13 +17,8 @@ pipeline:
...
     cmd: |
-      VERSION=$(git describe --tags --always)
-      if [[ $CDP_TARGET_BRANCH == master && ! $CDP_PULL_REQUEST_NUMBER ]]; then
-        IMAGE=registry-write.opensource.zalan.do/teapot/stackset-controller
-      else
-        IMAGE=registry-write.opensource.zalan.do/teapot/stackset-controller-test
-      fi
-      IMAGE=$IMAGE VERSION=$VERSION make build.push
+      IMAGE=registry-write.opensource.zalan.do/teapot/stackset-controller-test
+      IMAGE=$IMAGE VERSION=$CDP_BUILD_VERSION make build.push
```
Comment (aryszka) [inline delivery.yaml]: Question: why do we do `IMAGE=$IMAGE`? To have the value on a separate line for readability?

**Comment by aermakov-zalando** [comment]
👍

File: delivery.yaml (hunk 1 above)
Comment (szuecs) [reply delivery.yaml]: it was easier to copy and paste
I need faster iterations. I can clean it up later, if it's successful

**Pull request merged**